File: resources/expected/ros2/isolette/lax/src/isolette_cpp_pkg/src/base_code/operator_interface_oip_oit_runner.cpp
#include "isolette_cpp_pkg/user_headers/operator_interface_oip_oit_src.hpp"

//=================================================
//  D O   N O T   E D I T   T H I S   F I L E
//=================================================

operator_interface_oip_oit::operator_interface_oip_oit() : operator_interface_oip_oit_base()
{
    // Invoke initialize entry point
    initialize();

    PRINT_INFO("operator_interface_oip_oit infrastructure set up");
}

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto executor = rclcpp::executors::MultiThreadedExecutor();
    auto node = std::make_shared<operator_interface_oip_oit>();
    executor.add_node(node);
    executor.spin();
    rclcpp::shutdown();
    return 0;
}

File: resources/expected/ros2/python-isolette/strict/src/isolette_py_pkg/isolette_py_pkg/base_code/operator_interface_oip_oit_runner.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from rclpy.executors import MultiThreadedExecutor
from isolette_py_pkg.user_code.operator_interface_oip_oit_src import operator_interface_oip_oit
#========================================================
# Re-running Codegen will overwrite changes to this file
#========================================================
def main(args=None):
    rclpy.init(args=args)
    node = operator_interface_oip_oit()
    executor = MultiThreadedExecutor()
    executor.add_node(node)
    executor.spin()
    rclpy.shutdown()

if __name__ == "__main__":
    main()
